docs/release-notes/release-notes-0.15.0.md

```diff
@@ -92,6 +92,10 @@
   change, it allows encrypted failure messages to be returned to the sender.
   Additionally it is possible to signal a malformed htlc.
 
+* Add [SubscribePayments](https://github.com/lightningnetwork/lnd/pull/6335)
```

Comment:
Maybe nice to mention in the PR description some use cases for this new functionality.

Reply:
Done!

Reply:
Nice addition!